swupd/fullfiles.go

```diff
@@ -49,6 +49,12 @@ func CreateFullfiles(m *Manifest, chrootDir, outputDir string) error {
 			// if the file could be compressed.
 			output := filepath.Join(outputDir, name+".tar")
 
+			// Don't regenerate if file exists.
```

Comment:
Only if the file exists in a location that is user editable, or could have been a failed download/creation.

Reply:
Could you elaborate a bit on what cases we should skip or not?

To identify a failed creation, we need to verify the hash of the file. So we should just skip if the hash matches, and try to rewrite if it doesn't?